common/utils/src/main/resources/error/error-classes.json

```diff
@@ -143,6 +143,50 @@
       "Could not load Protobuf class with name <protobufClassName>. <explanation>."
     ]
   },
+  "CANNOT_LOAD_STATE_STORE" : {
+    "message" : [
+      "An error occurred during loading state."
+    ],
+    "subClass" : {
+      "CANNOT_READ_CHECKPOINT" : {
+        "message" : [
+          "Cannot read RocksDB checkpoint metadata of version <versionLine>"
```

Comment:
Can you provide some hits to users like what we expect. From the source code:
```scala
      if (versionLine != s"v$VERSION") {
        throw QueryExecutionErrors.cannotReadCheckpoint(versionLine)
      }
```

Reply:
I'm not sure I understand, do you mean changing the error message so it reads like "Cannot read RocksDB checkpoint metadata. Expected version <v$VERSION>, found <versionLine>." instead?

Reply:
> Expected version <v$VERSION>, found ." instead?

Yep, something like:
```
Expects the <expectedVersion> version, but found <actualVersion>.
```